api/report_viewer_server.thrift

```diff
@@ -279,6 +285,93 @@ service codeCheckerDBAccess {
                                              2: i64 new_run_id,
                                              3: DiffType diff_type,
                                              4: ReportFilterList reportFilters)
-                                             throws (1: shared.RequestFailed requestError)
+                                             throws (1: shared.RequestFailed requestError),
```

Comment:
:ballot_box_with_check: This has been done, just GitHub is not picking it up. :wink: